letsencrypt/cli.py

```diff
@@ -901,24 +922,15 @@ def _reconstitute(config, full_path):
         logger.debug("Traceback was:\n%s", traceback.format_exc())
         return None
 
-    # webroot_map is, uniquely, a dict, and the general-purpose
-    # configuration restoring logic is not able to correctly parse it
-    # from the serialized form.
-    if "webroot_map" in renewalparams and not _set_by_cli("webroot_map"):
-        setattr(config.namespace, "webroot_map", renewalparams["webroot_map"])
-
     try:
-        domains = [le_util.enforce_domain_sanity(x) for x in
-                   renewal_candidate.names()]
+        for d in renewal_candidate.names():
+            _process_domain(config, d)
```

Comment:
Are you sure that this correctly updates config in substantially the same way the old code did?